foil.tex

```diff
@@ -34,32 +30,28 @@
 %
 }
 
-%\date{2008.6.4}
-
 \begin{document}
 
 \maketitle
 
 \tableofcontents
 
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Do not make any visual bar to be longer than 79 characters because most standard terminals have 80 characters in a line.

Reply:
> Do not make any visual bar to be longer than 79 characters because most standard terminals have 80 characters in a line.

I changed the visual bar and content line length to 79 characters. However, equations and flowcharts don't follow this constraint.

Reply:
It's good.  I suppose you are ready for the next review.  Please remember to leave a global comment to request for it.